templates/e3sm_diags.bash

```diff
@@ -1,13 +1,12 @@
 #!/bin/bash
 {% include 'slurm_header.sh' %}
+{%- if use_dev_env -%}
+module load anaconda3/2019.03
+source /share/apps/anaconda3/2019.03/etc/profile.d/conda.sh
+conda activate e3sm_diags_env_dev
+{%- else -%}
 {% include 'e3sm_unified' %}sh
-
-# To load custom E3SM Diags environment, comment out line above using {# ... #}
-# and uncomment lines below
-
-#module load anaconda3/2019.03
-#source /share/apps/anaconda3/2019.03/etc/profile.d/conda.sh
-#conda activate e3sm_diags_env_dev
```

Comment:
These commands are fine for manually debugging an individual `e3sm_diags_*.bash` file,  but informing `zppy` of which environment to use is more difficult. Firstly, `unified` is needed to even run `zppy/post.py`. Additionally, loading a custom environment isn't enough -- we'd also have to `pip install` the changes we want to use from an `e3sm_diags` repo/directory. Lastly, anyone who develops `zppy` would have to use the `e3sm_diags_env_dev` environment going forward or else failures may result.